### ISSUE TITLE
why label sholud be [0 255]

### ISSUE THREAD

**Issue by zack6514** [sukritshankar/CaffeLMDBCreationMultiLabel#8]
I have 4 classes and assign 0 0 1 1 to it ， and in train.prototxt  label data not have the param scale , I think this is ok? why should be [0 255]?

**Comment by sukritshankar** [comment]
We are using data layer for labels. Data layer in Caffe automatically scales down by a factor of 255. Thats why labels should be in the scale of [0, 255], so that after scaling, you should have proper values between 0 and 1 !!

**Comment by vishalghor** [comment]
Hi @sukritshankar ,

Can you give more information on creating .mat file labels.
How to create the .mat file?
It would be helpful if you could help me for the same as i am new to caffe framework.

Thanks and Regards

**Comment by aniket03** [comment]
@sukritshankar I believe its the code in prototxt
https://github.com/sukritshankar/Caffe-LMDBCreation-MultiLabel/blob/f4a92fcf4282ea808cd3c6bb5ade2712e23d2dbf/train_vgg_11_sigmoid_cross_entropy_loss.prototxt#L43 due to which caffe will scale down the labels by 255. Thus if we eliminate this line of code should labels of the form [0, 1] not be directly useful. Without having to scale them up by 255.

**Comment by aniket03** [comment]
Hi, @sukritshankar any updated on above?

**Comment by iiwindii** [comment]
@aniket03 Agree! If your labels are 0 ,1, then it should work to eliminate this line.
